packages/zoe/src/objArrayConversion.js

```diff
@@ -81,3 +94,16 @@ export const filterFillAmounts = (allocation, amountMathKeywordRecord) => {
   });
   return filledAllocation;
 };
```

Comment:
`filterFillAmounts` is a really nice helper. However, this one really is specific to keyword records and even knows about amountMath, raising the issue of the intended generality of this file. Given the specificity, using `undefined` or falsyness as an absence check here is sound.

```suggestion
export const filterFillAmounts = (allocation, amountMathKeywordRecord) => {
  const filterFillPair = 
      ([keyword, amountMath]) => [key, allocation[keyword] || amountMath.getEmpty()];
  return objectMap(amountMathKeywordRecord, filterFillPair);
}
```

Reply:
removed function